scripts/exclude_tests.diff

```diff
@@ -102,12 +204,12 @@ index 25ead15772..f830d1c2ff 100644
 -                        ;; Timeseries drivers currently support only testing with pre-loaded dataset
 -                        (remove (tqpt/timeseries-drivers)))
 -    (mt/dataset (mt/dataset-definition "custom-pk"
--                  ["user"
--                   [{:field-name "custom_id" :base-type :type/Integer :pk? true}]
--                   [[1]]]
--                  ["group"
--                   [{:field-name "user_custom_id" :base-type :type/Integer :fk "user"}]
--                   [[1]]])
+-                                       ["user"
+-                                        [{:field-name "custom_id" :base-type :type/Integer :pk? true}]
+-                                        [[1]]]
+-                                       ["group"
+-                                        [{:field-name "user_custom_id" :base-type :type/Integer :fk "user"}]
+-                                        [[1]]])
```

Comment:
This is the `(deftest dataset-with-custom-pk-test` test, judging by its name it sounds like should be excluded by default for drivers with `:metadata/key-constraints`?